.github/workflows/cargo-doc.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
       - run: cargo doc
 
       - name: Deploy
-        if: ${{ github.ref == 'ref/head/master' }}
+        if: ${{ github.ref == 'refs/heads/soares/nary-test-chains' }}
```

Comment:
Let's revert this before merging?

Reply:
Right, actually I keep having problem finding the correct expression to trigger this step. It is not triggering on master as well.

Reply:
I have reverted the branch to master. It probably still won't work on master, but we can figure out a fix in a separate PR.